google-api-client/OVERVIEW.md

```diff
@@ -1,36 +1,45 @@
-# Google API Client Library for Ruby Docs
+# Monolithic Client for Google APIs (DEPRECATED)
 
-The Google API Client Library for Ruby is designed for Ruby
-client-application developers. It offers simple, flexible
-access to many Google APIs.
+This library includes REST clients for all Discovery-based Google APIs as of December 2020.
 
-## Features
+This library is **deprecated** and will likely not be updated in the future. Instead, please install the gem corresponding to the specific service you want to use. For example, to use the Google Drive V3 client, install the `google-apis-drive_v3` gem.
 
-- Call Google APIs simply
-- Handle Auth with fewer lines of code
-- Use standard tooling for installation
+## FAQ
 
-## Documentation
+### Why is this gem deprecated?
 
-Learn how to use the Google API Client Library for Ruby with these guides:
+The `google-api-client` gem is a 75 megabyte installation including client classes for more than 300 services. Although the library is designed so that only the classes being used are loaded into memory, it still takes a nontrivial amount of space on disk (and in deployments), and can take a while to install. Additionally, the documentation is so large that rubydoc.info often fails to build it.
```

Comment:
If the documentation is removed, should this be past tense? ("was so large... failed..")

Reply:
The class reference documentation is still there. (Only the handwritten usage documentation was removed.) Updated to clarify.